sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/core/ReactiveCosmosTemplateIT.java

```diff
@@ -281,6 +296,36 @@ public void testUpsert() {
         assertThat(responseDiagnosticsTestUtils.getCosmosDiagnostics()).isNotNull();
     }
 
+    @Test
+    public void testPatch() {
+        cosmosTemplate.patch(insertedPerson, operations, null);
```

Comment:
Not sure how these tests are passing, but patch operation through ReactiveCosmosTemplate is an async operation and will not execute unless subscribed to it. 
Same for other tests in this class. 
Please revisit the testing and let's make sure we have good and correct coverage on this, thanks!

Reply:
Done.